### ISSUE TITLE
Add Confidence Level Arbitration to the Test Execution sequence diagram example

### ISSUE THREAD

**Issue by e-backmark-ericsson** [Ericsson/eiffel#186]
### Description
In the current version of the [usage example for Test Execution](https://github.com/Ericsson/eiffel/blob/master/usage-examples/test-execution.md) the CLM event is sent from Test Orchestrator. The sequence diagram actors should follow the pipeline actors defined by [Sepia](https://eiffel-community.github.io/eiffel-sepia/), once it has been set, and therefore we should add a Confidence Level Arbitration actor to the diagram and make it responsible of sending the CLM event. At the same time the current actors should be renamed according to Sepia.

### Motivation
To align our usage examples with Sepia

### Exemplification
Move the CLM event sending from the Test Execution actor to the new actor Confidence Level Arbitration

### Benefits
It would be aligned with our proposed implementation architecture, Sepia.

### Possible Drawbacks
Additional actors in the picture might scare people. To mitigate that the description in the example should state that a realization of the sequence diagram need not contain all those actors as separate processes/services, but could very well be co-located in fewer components.

**Comment by d-stahl-ericsson** [comment]
I think it's better to leave the sequence diagram out of this repository, and keep that in Sepia instead. I added an adaption of it to the [Test Orchestration](https://eiffel-community.github.io/eiffel-sepia/test-orchestration.html) page. We could add Confidence Level Arbitration there. What do you think?

**Comment by e-backmark-ericsson** [comment]
You're right. It is better suited in Sepia. Once Sepia is in good shape we should instead remove the sequence diagram from this example.
